core/src/main/java/com/linecorp/armeria/server/docs/DocServiceInjectableScripts.java

```diff
@@ -0,0 +1,194 @@
+/*
+ *  Copyright 2025 LY Corporation
+ *
+ *  LY Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+package com.linecorp.armeria.server.docs;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+import java.util.regex.Pattern;
+
+import com.linecorp.armeria.common.annotation.UnstableApi;
+
+/**
+ * Provides utilities for {@link DocServiceBuilder#injectedScripts(String...)}.
+ */
+@UnstableApi
+public final class DocServiceInjectableScripts {
```

Comment:
Would you also add the link to this class to `DocServiceBuilder.injectedScripts`?
```
@see DocServiceInjectableScripts for predefined scripts that can be injected.
```